addons/web/static/src/js/components/action_menus.js

```diff
@@ -30,59 +42,65 @@ odoo.define('web.ActionMenus', function (require) {
         }
 
         //---------------------------------------------------------------------
-        // Getters
+        // Private
         //---------------------------------------------------------------------
 
         /**
-         * @returns {Object[]}
+         * Add teh tooltips to the items
+         * @private
+         */
+        _addTooltips() {
+            $(this.el.querySelectorAll('[title]')).tooltip({
+                delay: { show: 500, hide: 0 }
+            });
+        }
+
+        /**
+         * @private
+         * @param {Object} props
+         * @returns {Promise<Object[]>}
          */
-        get actionItems() {
+        async _setActionItems(props) {
             // Callback based actions
-            const callbackActions = (this.props.items.other || []).map(
+            const callbackActions = (props.items.other || []).map(
                 action => Object.assign({ key: `action-${action.description}` }, action)
             );
             // Action based actions
-            const actionActions = this.props.items.action || [];
-            const relateActions = this.props.items.relate || [];
+            const actionActions = props.items.action || [];
+            const relateActions = props.items.relate || [];
             const formattedActions = [...actionActions, ...relateActions].map(
                 action => ({ action, description: action.name, key: action.id })
             );
             // ActionMenus action registry components
-            const registryActions = this.constructor.registry.values().map(
-                ({ Component, getProps }, index) => ({
-                    key: `registry-action-${index}`,
-                    Component,
-                    props: getProps(this.props),
-                })
-            );
+            const registryActions = [];
+            const rpc = this.rpc.bind(this);
+            for (const { Component, getProps } of this.constructor.registry.values()) {
+                const itemProps = await getProps(props, this.env, rpc);
```

Comment:
falsy value means we should not render the item. I'm not found of this. However, I understand that we are constrained here as we don't want to adapt all getProps functions. In your version of this fix for master, we'll need to find a better API (if this is still relevant)